src/goto-instrument/unwindset.cpp

```diff
@@ -21,43 +21,52 @@ void unwindsett::parse_unwind(const std::string &unwind)
     global_limit = unsafe_string2unsigned(unwind);
 }
 
-void unwindsett::parse_unwindset(const std::string &unwindset)
+void unwindsett::parse_unwindset_one_loop(std::string val)
 {
-  std::vector<std::string> unwindset_loops =
-    split_string(unwindset, ',', true, true);
+  unsigned thread_nr = 0;
+  bool thread_nr_set = false;
 
-  for(auto &val : unwindset_loops)
+  if(!val.empty() && isdigit(val[0]) && val.find(":") != std::string::npos)
```

Comment:
Wouldn't this also be true for `0c::main.1`?

Reply:
yes -- but seems unrelated?